### ISSUE TITLE
Cancelled events are not displayed in an accessible way in the calendar grid

### ISSUE THREAD

**Issue by GVodyanov** [nextcloud/calendar#7932]
### Steps to reproduce

Currently cancelled events are shown with a strikethrough. Screenreaders however do not convey this information, leading to a loss of information.

### Expected behavior

A way to fix this could be to add invisible text saying "Cancelled" next to the title of the event

### Actual behavior

Only strikethrough, not accessible

### Web server

None

### Database engine version

None

### PHP engine version

None

### Updated from an older installed version or fresh install

None

### List of activated apps

```Text

```

### Nextcloud configuration

```Text

```

### Web server error log

```Text

```

### Log file

```Text

```

### Browser log

```Text

```

**Comment by GVodyanov** [comment]
Although it does have a title saying you declined this event..?

**Comment by GVodyanov** [comment]
After some testing with a screen reader: the title on cancelled events is picked up on by a screen reader, making that part not a problem

What is a more significant problem is that the fullcalendar grid in general really doesn't work that well... Orca doesn't seem to pick up on event descriptions at all, only voicing the html title. Hence, I will be renaming this ticket to reflect the changes that need to be done to the fullcalendar grid.

**Comment by SebastianKrupinski** [comment]
Can this be closed then?
